diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/RulesConfigYamlTest.kt

```diff
@@ -36,6 +38,16 @@ inline class RulesConfigYamlTest(private val pathMap: Map<String, String> =
         checkComments("../diktat-analysis.yml")
     }
 
+    @Test
+    fun `check kotlin version`() {
+        val currentKotlinVersion = KotlinVersion.CURRENT
+        pathMap.keys.forEach { path ->
+            val config = readAllRulesFromConfig(path)
+            val ktVersion = config.find { it.name == DIKTAT_COMMON }?.configuration?.get("kotlinVersion")?.kotlinVersion()
```

Comment:
```suggestion
            val ktVersion = config.getCommonConfiguration().kotlinVersion
```
You already have this logic wrapped in convenient methods, use them.

Reply:
I cannot use this logic, since I need to check that the parameter is read correctly and is not replaced by the default value